bin/utils/convert-argv.js

```diff
@@ -104,14 +104,29 @@ module.exports = function(...args) {
 
 		const requireConfig = function requireConfig(configPath) {
 			let options = (function WEBPACK_OPTIONS() {
+				const resolvedPath = path.resolve(process.cwd(), configPath);
 				if (argv.configRegister && argv.configRegister.length) {
 					module.paths.unshift(path.resolve(process.cwd(), "node_modules"), process.cwd());
 					argv.configRegister.forEach(dep => {
 						require(dep);
 					});
-					return require(path.resolve(process.cwd(), configPath));
+					return require(resolvedPath);
 				} else {
-					return require(path.resolve(process.cwd(), configPath));
+					try {
+						return require(resolvedPath);
+					} catch (err) {
+						if (err.stack.indexOf("ERR_REQUIRE_ESM") >= 0) {
+							if (process.execArgv.indexOf("--experimental-modules") < 0) {
```

Comment:
```suggestion
							if (process.execArgv.includes("--experimental-modules")) {
```

Reply:
I don't think that is a good idea as we're still supporting old node versions at master

Reply:
Just fyi, we are supporting up to Node v8, and support for Array `.includes()` is since v6.
According to MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

Reply:
Thanks for elaborating. Changing